components/lock.rst

```diff
@@ -427,6 +427,29 @@ You can also create this table explicitly by calling the
 :method:`Symfony\\Component\\Lock\\Store\\PdoStore::createTable` method in
 your code.
 
+.. _lock-store-pgsql:
+
+PostgreSqlStore
+~~~~~~~~~~~~~~~
+
+The PostgreSqlStore uses `Advisory Locks`_ provided by Postgresql. It requires a
+ `PDO`_ connection, a `Doctrine DBAL Connection`_, or a
+ `Data Source Name (DSN)`_. it nativly supports blocking, as weel as sharing
+ locks.
+
+    use Symfony\Component\Lock\Store\PostgreSqlStore;
+
+    // a PDO, a Doctrine DBAL connection or DSN for lazy connecting through PDO
+    $databaseConnectionOrDSN = 'postgresql://myuser:mypassword@localhost:5634/lock';
+    $store = new PostgreSqlStore($databaseConnectionOrDSN);
+
+In opposite to the `PdoStore`, the `PostgreSqlStore` does not need a table to stores
```

Comment:
Please, use double backticks to enclose code:

```rst
[...] `PdoStore`, the `PostgreSqlStore` does [...]

-->

[...] ``PdoStore``, the ``PostgreSqlStore`` does [...]
```

Reply:
thanks